src/redux/reducers/componentConfig/initialState.js

```diff
@@ -71,7 +71,9 @@ const embeddingContinuousInitialConfig = {
   marker: markerBaseState,
   labels: labelBaseState,
   logEquation: 'datum.expression*1',
-  shownGene: 'notSelected',
```

Comment:
I replaced this `notSelected` value with null because it will never be seen: if the component detects this value it shows loading until it displays the default gene. This means that the value is performing the same function as `null` usually does, so I replaced it with `null`